python/ray/data/_internal/execution/resource_manager.py

```diff
@@ -479,7 +484,9 @@ def _update_reservation(self):
             min_reserved.object_store_memory += self._reserved_for_op_outputs[op]
             # Total resources we want to reserve for this operator.
             op_total_reserved = default_reserved.max(min_reserved)
-            if op_total_reserved.satisfies_limit(self._total_shared):
+            if op_total_reserved.satisfies_limit(
```

Comment:
Please add a comment elaborating on 

 - That reserving min for at least 1 operator guarantees liveness
 - That we're ignoring object store task requirement b/c task scheduling doesn't consider Object Store

Reply:
added comments